src/feature/relay/router.c

```diff
@@ -372,6 +372,8 @@ assert_identity_keys_ok(void)
   }
 }
 
+#ifdef HAVE_MODULE_RELAY
```

Comment:
This is okay for now, though eventually we should make this whole file relay-only.

Reply:
I opened tickets to:
* fix router.c - https://trac.torproject.org/projects/tor/ticket/32162
* fix similar issues with the dirauth module - https://trac.torproject.org/projects/tor/ticket/32163